project1/styles.css

```diff
@@ -0,0 +1,85 @@
+body {
+    color: rgb(85, 85, 85);
+    background-color: rgb(230, 230, 230);
+    font-family: 'Cormorant Garamond', serif; 
+    
+ 
+     
+}
+
+.wrapper {
```

Comment:
Since your wrapper class doesn't have its own background color, I would recommend simplifying this by using _either_ margins or padding, but not both. What you have now isn't wrong or broken, but it's more complicated than it needs to be for this instance.

If you want the wrapper class to be centered at all times, I would recommend:
```
.wrapper {
    margin: 10px auto;
    width: 80%;
}
```